tools, readme: adjust Lightning docs for the latest version (#799)
pingcap/docs: tools/lightning/checkpoints.md

# The schema name (database name) to store the checkpoints
schema = "tidb_lightning_checkpoint"

# Where to store the checkpoints.
#  - file:  store as a local file.
#  - mysql: store into a remote MySQL-compatible database
driver = "file"

# The data source name (DSN) indicating the location of the checkpoint storage.
#

> # For "file" driver, the DSN is a path. If not specified, Lightning would

Reviewer: The subject of "If not specified" is the path, while the subject of the latter part is Lightning. It's inappropriate in grammar to omit it when the two are inconsistent. When they are consistent, you can omit it:)

# default to "/tmp/CHECKPOINT_SCHEMA.pb".
#

> # For "mysql" driver, the DSN is a URL in the form of "USER:PASS@tcp(HOST:PORT)/".

Reviewer: ```suggestion
# For the "mysql" driver, the DSN is a URL in the form of "USER:PASS@tcp(HOST:PORT)/".
```

> # If not specified, the TiDB server from the [tidb] section will be used to

Reviewer: ```suggestion
# If the URL is not specified, the TiDB server from the [tidb] section is used to
```

## Checkpoints storage

Lightning supports two kinds of checkpoint storage: a local file or a remote MySQL-compatible database.

> With `driver = "file"`, checkpoints are stored in a local file at the path given by the `dsn` setting. Checkpoints are updated rapidly, and we highly recommend to place the checkpoint file on a drive with very high write endurance e.g. a RAM disk.

Reviewer: ```suggestion
With `driver = "file"`, checkpoints are stored in a local file at the path given by the `dsn` setting. Checkpoints are updated rapidly, so we highly recommend placing the checkpoint file on a drive with very high write endurance, such as a RAM disk.
```

> With `driver = "mysql"`, checkpoints can be saved in any databases compatible with MySQL 5.7 or above, including MariaDB and TiDB. By default, the checkpoints are saved in the target database.

Reviewer: For the two paras above, it would be clearer if we use a bulleted list.

> With `driver = "mysql"`, checkpoints can be saved in any databases compatible with MySQL 5.7 or above, including MariaDB and TiDB. By default, the checkpoints are saved in the target database.

Reviewer: ```suggestion
With `driver = "mysql"`, checkpoints can be saved in any databases compatible with MySQL 5.7 or later, including MariaDB and TiDB. By default, the checkpoints are saved in the target database.
```
